packages/ngtools/webpack/src/paths-plugin.ts

```diff
@@ -155,39 +159,61 @@ export class TypeScriptPathsPlugin {
         // to be created.
         const replacements = findReplacements(originalRequest, this.patterns);
 
-        const tryResolve = () => {
+        const attemptResolveRequest = (request: DoResolveValue): Promise<DoResolveValue | null> => {
+          return new Promise((resolve, reject) => {
+            resolver.doResolve(
+              target,
+              request,
+              '',
+              resolveContext,
+              (error: Error | null, result: DoResolveValue) => {
+                if (error) {
+                  reject(error);
+                } else if (result) {
+                  resolve(result);
+                } else {
+                  resolve(null);
+                }
+              },
+            );
+          });
+        };
+
+        const tryNextReplacement = () => {
           const next = replacements.next();
           if (next.done) {
             callback();
 
             return;
           }
 
-          const potentialRequest = {
+          // Resolution in the target location, preserving the original request.
+          // This will work with the `resolve-in-package` resolution hook, supporting
+          // package exports for e.g. locally-built APF libraries.
+          const potentialRequestAsPackage = {
+            ...request,
+            path: path.resolve(this.baseUrl ?? '', next.value),
```

Comment:
Nit: you can store `path.resolve(this.baseUrl ?? '', next.value),` in a const sine you are accessing it multiple times